tweets/2019/04/mlbot.tweet

```diff
@@ -0,0 +1,3 @@
+How to Automate Tasks on GitHub With Machine Learning for Fun and Profit (using Python)
```

Comment:
Maybe something like this?
```suggestion
A friend of Probot wrote an article on how to use Python to Automate Tasks on GitHub With Machine Learning!
```
I think just saying "using python" could still be ambiguous: "Wait you can use Probot with Python??"

Reply:
Are they a friend of Probot? Do you know them?

Reply:
No just trying to lend some personality to it. How about

```suggestion
Look at what one of my cousins can do: Using Python to Automate Tasks on GitHub With Machine Learning!
--



```"Look at what one of my cousins can do!?

Reply:
I find it a little weird 🤔 I think it might work if probot would talk about a similar  project, like a Python-based GitHub App framework. 

Maybe we could just prefix the text with "📰 Interesting read: "

> 📰 Interesting read: How to Automate Tasks on GitHub With Machine Learning for Fun and Profit #Python

Or something like that?
 

Reply:
Yeh that looks good to me. Simple and makes it clear that it's just something we're interested in.

Reply:
Ok, I updated the tweet.